<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>444</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="mainWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>451</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btnSend">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>390</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Отправить</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblUserName">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>0</y>
       <width>391</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="te_Message">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="chatText">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>401</width>
       <height>351</height>
      </rect>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="NameWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>381</height>
     </rect>
    </property>
    <widget class="QTextEdit" name="te_Name">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnName">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>190</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Войти в чат</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblName">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>151</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Введите ваше имя:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
